yarn-project/end-to-end/src/e2e_account_contract.test.ts

```diff
@@ -80,50 +87,46 @@ describe('e2e_account_contract', () => {
   it('calls a private function', async () => {
     await deployL2Contracts();
     logger('Calling private function...');
-    const tx1 = child.methods.value(42).send({ from: schnorrAccountContract.address });
-    const tx2 = child.methods.value(53).send({ from: ecdsaAccountContract.address });
+    const tx1 = child.methods.value(42).send({ from: schnorrAccountContractAddress });
 
-    const txs = [tx1, tx2];
+    const txs = [tx1];
 
     await Promise.all(txs.map(tx => tx.isMined(0, 0.1)));
     const receipts = await Promise.all(txs.map(tx => tx.getReceipt()));
 
     expect(receipts[0].status).toBe(TxStatus.MINED);
-    expect(receipts[1].status).toBe(TxStatus.MINED);
   }, 60_000);
 
-  it('calls a public function', async () => {
-    await deployL2Contracts();
-    logger('Calling public function...');
-    const tx1 = child.methods.pubStoreValue(42).send({ from: schnorrAccountContract.address });
-    const tx2 = child.methods.pubStoreValue(53).send({ from: ecdsaAccountContract.address });
-
-    const txs = [tx1, tx2];
-
-    await Promise.all(txs.map(tx => tx.isMined(0, 0.1)));
-    const receipts = await Promise.all(txs.map(tx => tx.getReceipt()));
-
-    expect(receipts[0].status).toBe(TxStatus.MINED);
-    expect(receipts[1].status).toBe(TxStatus.MINED);
-    // The contract accumulates the values so the expected value is 95
-    expect(toBigInt((await aztecNode.getStorageAt(child.address, 1n))!)).toEqual(95n);
-  }, 60_000);
-
-  it('fails to execute function with invalid schnorr signature', async () => {
-    logger('Registering ecdsa signer against schnorr account contract');
-    // Set the incorrect signer for schnorr
-    await deployL2Contracts(CurveType.GRUMPKIN, SignerType.ECDSA, CurveType.SECP256K1, SignerType.ECDSA);
-    await expect(child.methods.value(42).create({ from: schnorrAccountContract.address })).rejects.toMatch(
-      /could not satisfy all constraints/,
-    );
-  }, 60_000);
-
-  it('fails to execute function with invalid ecdsa signature', async () => {
-    logger('Registering schnorr signer against ecdsa account contract');
-    // Set the incorrect signer for ecdsa
-    await deployL2Contracts(CurveType.GRUMPKIN, SignerType.SCHNORR, CurveType.SECP256K1, SignerType.SCHNORR);
-    await expect(child.methods.value(42).create({ from: ecdsaAccountContract.address })).rejects.toMatch(
-      /could not satisfy all constraints/,
-    );
-  }, 60_000);
+  // it('calls a public function', async () => {
```

Comment:
We should uncomment or delete these tests. I believe we should be ok deleting the happy paths, since they are effectively tested in the other e2e suites, but we should still test the failing signature getting rejected.